mithril-signer/scripts/verify_signer_signature.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$AGGREGATOR_ENDPOINT" ] || [ -z "$PARTY_ID" ]; then
+    echo ">> ERROR: Required environment variables AGGREGATOR_ENDPOINT and/or PARTY_ID are not set."
+    exit 1
+fi
+
+curl -s "$AGGREGATOR_ENDPOINT/certificates" -H 'accept: application/json' | jq -r '.[] | .hash' | while read -r HASH; do
+    RESPONSE=$(curl -s "$AGGREGATOR_ENDPOINT/certificate/$HASH" -H 'accept: application/json')
+    SIGNER_COUNT=$(echo "$RESPONSE" | jq '.metadata.signers | length')
+    for (( i=0; i < SIGNER_COUNT; i++ )); do
+        PARTY_ID_RESPONSE=$(echo "$RESPONSE" | jq -r ".metadata.signers[$i].party_id")
+        if [[ "$PARTY_ID_RESPONSE" == "$PARTY_ID" ]]; then
+            echo ">> You have signed this certificate : $HASH"
+            exit 1
+        fi
+    done
+done
+
+echo ">> Your party id was not found in the last certificates"
```

Comment:
```suggestion
echo ">> Oops, your party id was not found in the last ${SIGNER_COUNT} certificates. Please try again later."
```

Reply:
Ok, I just need to use another variable than ${SIGNER_COUNT} to count certificates